templates/advantage/_stripe-modal.html

```diff
@@ -172,27 +172,29 @@ <h2 class="p-modal__title js-contract-name u-no-margin--bottom" id="modal-title"
         <i class="p-icon--spinner u-animation--spin"></i>
       </div>
     </div>
-    
-    <footer class="p-modal__footer u-align--right">
+
+    <footer class="p-modal__footer">
       <span id="js-progress-indicator" class="p-modal__progress u-hide">
         <i class="p-icon--success u-hide"></i>
         <i class="p-icon--spinner u-animation--spin"></i>&nbsp;&nbsp;
         <span></span>
       </span>
 
-      <button class="js-cancel-modal" style="min-width: 5.5rem;">
-        Cancel
-      </button>
+      <div class="row u-no-padding">
+        <button class="js-cancel-modal col-small-2 col-medium-3 col-3" style="text-align: center;">
```

Comment:
```suggestion
        <button class="js-cancel-modal col-small-2 col-medium-2 col-start-medium-3 col-start-large-7 col-3 u-no-margin--right" style="text-align: center;">
```

- `col-medium-2` stops the buttons from being huge at medium screen sizes.
- `col-start-` restores the right alignment.
- `u-no-margin--right` restores the proper Vanilla spacing between buttons…